envs/scIB-python.yml

```diff
@@ -5,508 +5,63 @@ channels:
-  - tzlocal=2.0.0=py_0
-  - umap-learn=0.3.9=py37_0
-  - wcwidth=0.1.7=py_1
-  - webencodings=0.5.1=py_1
-  - werkzeug=0.15.5=py_0
-  - wheel=0.33.4=py37_0
-  - widgetsnbextension=3.5.0=py37_0
-  - wrapt=1.11.2=py37h7b6447c_0
-  - xorg-kbproto=1.0.7=h14c3975_1002
-  - xorg-libice=1.0.10=h516909a_0
-  - xorg-libsm=1.2.3=h84519dc_1000
-  - xorg-libx11=1.6.8=h516909a_0
-  - xorg-libxau=1.0.9=h14c3975_0
-  - xorg-libxdmcp=1.1.3=h516909a_0
-  - xorg-libxext=1.3.4=h516909a_0
-  - xorg-libxrender=0.9.10=h516909a_1002
-  - xorg-renderproto=0.11.1=h14c3975_1002
-  - xorg-xextproto=7.3.0=h14c3975_1002
-  - xorg-xproto=7.0.31=h14c3975_1007
-  - xz=5.2.4=h14c3975_1001
-  - yaml=0.1.7=had09818_2
-  - zeromq=4.3.2=he1b5a44_2
-  - zipp=0.5.2=py_0
-  - zlib=1.2.11=h516909a_1005
-  - zstd=1.4.0=h3b9ef0a_0
+  - numpy
+  - pandas
+  - seaborn
+  - matplotlib
+  - scanpy==1.4.6
+  - scipy
+  - memory_profiler
+  - rpy2>=3
+  - anndata2ri
+  - bbknn==1.3.9
+  - libgcc-ng
+  - scikit-learn
+  - networkx
+  - r-base
+  - r-seurat==3.0.2
+  - bioconductor-scater
+  - bioconductor-scran
+  - pip
+  - numba<=0.46
+  - llvmlite
+  - tensorflow==1.15
+  - gxx_linux-64
+  - gxx_impl_linux-64
+  - gcc_linux-64
+  - gfortran_linux-64
+  - icu
+  - GraphViz
+  - igraph
+  - openblas
+  - r-essentials
+  - r-globals==0.12.5
+  - r-listenv==0.8.0
+  - r-rlang
+  - r-ellipsis
+  - r-evaluate
+  - r-farver
+  - r-fnn
+  - r-isoband
+  - r-lifecycle
+  - r-praise
+  - r-testthat
+  - r-vctrs
+  - xlrd
+  - umap-learn==0.3.10
+  - louvain==0.6.1
+  - scvi
   - pip:
-    - absl-py==0.7.1
-    - adjusttext==0.7.3
     - anndata==0.7.1
-    - anndata2ri==1.0.2
-    - annoy==1.16.0
-    - astor==0.8.0
-    - atomicwrites==1.3.0
-    - bbknn==1.3.5
-    - chardet==3.0.4
-    - cython==0.29.13
-    - fa2==0.3.5
-    - fbpca==1.0
-    - future==0.17.1
-    - future-fstrings==1.2.0
-    - fuzzywuzzy==0.17.0
-    - gast==0.2.2
-    - get-version==2.0.6
-    - gprofiler-official==1.0.0
-    - grpcio==1.22.0
-    - h5py==2.10.0
-    - hyperopt==0.1.2
-    - idna==2.8
-    - intervaltree==2.1.0
-    - jupyter==1.0.0
-    - jupyterlab==1.0.5
-    - keras==2.2.4
-    - keras-vggface==0.6
-    - loompy==2.0.17
-    - matplotlib==3.0.3
-    - memory-profiler==0.55.0
-    - mnnpy==0.1.9.5
-    - more-itertools==7.2.0
-    - multicoretsne==0.1
-    - opencv-python==4.1.1.26
-    - packaging==19.0
-    - pillow==6.1.0
-    - pluggy==0.12.0
-    - protobuf==3.9.0
-    - psutil==5.6.3
-    - py==1.8.0
-    - pycparser==2.19
-    - pymongo==3.9.0
-    - pytest==5.0.1
-    - python-levenshtein==0.12.0
-    - pyyaml==5.1.2
-    - requests==2.22.0
-    - rpy2==3.0.5
-    - scanorama==1.4
-    - scanpy==1.4.5.dev114+gd69832a
-    - scgen==1.1.3
-    - scikit-learn==0.22.1
-    - scipy==1.2.1
-    - scvi==0.5.0
-    - sklearn==0.0
-    - sortedcontainers==2.1.0
-    - termcolor==1.1.0
-    - torch==1.2.0
-    - torchvision==0.5.0
-    - trvae==0.0.1
+    - trvae==1.1.2
     - trvaep==0.1.0
```

Comment:
can this be installed? Also, trvae 1.1.2 is installable, but we can't use it without the latest PR.